images/macos/scripts/build/install-common-utils.sh

```diff
@@ -35,19 +35,27 @@ for package in $cask_packages; do
 done
 
 # Load "Parallels International GmbH"
-if is_Monterey; then
+if is_Monterey || is_Sonoma || is_Ventura; then
     sudo kextload /Applications/Parallels\ Desktop.app/Contents/Library/Extensions/10.9/prl_hypervisor.kext || true
 fi
 
-# Execute AppleScript to change security preferences
+# Execute AppleScript to change security preferences for macOS12, macOS13 and macOS14
 # System Preferences -> Security & Privacy -> General -> Unlock -> Allow -> Not now
-if is_Monterey; then
+if is_Monterey || is_Sonoma || is_Ventura; then
     for retry in {4..0}; do
         echo "Executing AppleScript to change security preferences. Retries left: $retry"
         {
             set -e
             osascript -e 'tell application "System Events" to get application processes where visible is true'
-            osascript $HOME/utils/confirm-identified-developers.scpt $USER_PASSWORD
+            if is_Monterey; then
+                osascript $HOME/utils/confirm-identified-developers.scpt $USER_PASSWORD
+            else
+                if is_Ventura; then
+                    osascript $HOME/utils/confirm-identified-developers-macos.scpt $USER_PASSWORD "Ventura"
+                else
```

Comment:
Lets make a list here.
```
if is_Monterey; then
...
if is_Ventura; then
... "Ventura"
if is_Sonoma; then
... "Sonoma"
```
It will still crash if there is no match (should not happen, see the string 44), but it will be a little more transparent and convenient when another OS appears.

Reply:
Done 👍🏼